doc/snippets/Microsoft.Data.SqlClient/SqlClientPermission.xml

```diff
@@ -1,153 +1,166 @@
+      </returns>
+      <remarks>
+        Custom code that extends security objects must implement the <see cref="M:System.Data.Common.DBDataPermission.ToXml" /> and <see cref="M:System.Data.Common.DBDataPermission.FromXml" /> methods to make the objects security encodable.
+      </remarks>
+    </ToXml>
+    <FromXml>
+      <param name="securityElement">
+        The XML encoding to use to reconstruct the security object.
+      </param>
+      <summary>
+        Reconstructs a security object with a specified state from an XML encoding.
+      </summary>
+      <remarks>
+        Custom code that extends security objects needs to implement the <see cref="M:System.Data.Common.DBDataPermission.ToXml" /> and <see cref="M:System.Data.Common.DBDataPermission.FromXml" /> methods to make the objects security encodable.
+      </remarks>
+    </FromXml>
+    <Union>
+      <param name="target">
+        A permission object to combine with the current permission object. It must be of the same type as the current permission object.
+      </param>
+      <summary>
+        Returns a new permission object that is the union of the current and specified permission objects.
+      </summary>
+      <returns>
+        A new permission object that represents the union of the current permission object and the specified permission object.
+      </returns>
+      <remarks>
+          The result of a call to <see cref="M:System.Data.Common.DBDataPermission.Union" /> is a permission that represents all the operations permitted by both the current permission object and the specified permission object. Any demand that passes either permission passes their union.
+      </remarks>
+      <exception cref="T:System.ArgumentException">
+        The <paramref name="target" /> object is not the same type as the current permission object.
+      </exception>
+    </Union>
+    <IsSubsetOf>
+      <param name="target">
+        A permission object that is to be tested for the subset relationship. This object must be of the same type as the current permission object.
+      </param>
+      <summary>
+        Returns a value indicating whether the current permission object is a subset of the specified permission object.
+      </summary>
+      <returns>
+        <see langword="true" /> if the current permission object is a subset of the specified permission object, otherwise <see langword="false" />.
+      </returns>
+      <remarks>
+        The current permission object is a subset of the specified permission object if the current permission object specifies a set of operations that is wholly contained by the specified permission object. For example, a permission that represents access to C:\example.txt is a subset of a permission that represents access to C:\\. If this method returns <see langword="true" />, the current permission object represents no more access to the protected resource than does the specified permission object.
```

Comment:
Wondering if it's going to be safe to use C:\\ and C:\example.txt in text? Can you make them string literals by using double quotes?

Reply:
This is XML, so `\` doesn't have any special powers of escaping. If it were a character that needs escaping, it would have to be converted to an entity (eg, `&lt;`). But since it's a special string, I'll wrap it in a `<c></c>` to show it as code.